src/assets/_project/_blocks/components/forms/_qg-forms.scss

```diff
@@ -204,6 +204,7 @@
   .form-control.dropdown {
     padding: 8px;
     border-radius: $borderRadius;
+    height: 44px;
```

Comment:
Why this specific figure?

Reply:
https://www.qld.gov.au/law/assets/forms/formio-test/vaq/apply-for-financial-assistance
All the text/date input elements on prod has height of 44px when you inspect. Most of them is resulted by lin-height: 1.5. For dropdowns in particular, when it does not have a selected value, the height is 44px, when you select an item it goes 50px.